creator-node/src/routes/tracks.js

```diff
@@ -88,6 +88,9 @@ module.exports = function (app) {
     // exclude 0-length segments that are sometimes outputted by ffmpeg segmentation
     trackSegments = trackSegments.filter(trackSegment => trackSegment.duration)
 
+    // error if there are no track segments
+    if (!trackSegments || !trackSegments.length) return errorResponseServerError('Track upload failed - no track segments')
```

Comment:
why remove this? do we not want to error on empty track upload?

Reply:
@SidSethi remove what?

Reply:
brainfart, casually deleting